cmd/rollkit/commands/run_node.go

```diff
@@ -115,6 +112,16 @@ func NewRunNodeCmd() *cobra.Command {
 			// initialize the metrics
 			metrics := rollnode.DefaultMetricsProvider(cometconf.DefaultInstrumentationConfig())
 
+			// use mock jsonrpc da server by default
+			if !cmd.Flags().Lookup("rollkit.da_address").Changed {
+				srv, err := startMockDAServJSONRPC(context.Background())
+				if err != nil {
+					return fmt.Errorf("failed to launch mock da server: %w", err)
+				}
+				// nolint:errcheck,gosec
+				defer func() { srv.Stop(context.Background()) }()
```

Comment:
This works because there's no deadline, but I think ideally we'd want different contexts here so that a a cancel/deadline on one doesn't affect the other. e.g. `Start` times out, then we'd still want to call `Stop`.

Reply:
hmm, ok, but in what cases would you have an issue with start and not error out? 
Stop should be able to be called if the ctx timed out anyways i would think. 

It isn't a huge deal, since there are neither cancels or timeouts with these, so happy to resolve. 
I think this just illustrates our overall inconsistent use of context in this codebase 🙃